rules-reviewed/technology-usage/embedded-framework-technology-usage.windup.xml

```diff
@@ -1050,18 +1050,17 @@
                 </technology-identified>
             </perform>
         </rule>
-        <rule id="embedded-framework-embedded-framework-08950">
+        <rule id="technology-usage-embedded-framework-09000">
             <when>
                 <graph-query discriminator="TechnologyTagModel">
-                    <property name="name">Redis</property>
+                    <property name="name">Feign</property>
                 </graph-query>
             </when>
             <perform>
-                <technology-identified name="Redis">
-                    <tag name="Store"/>
+                <technology-identified name="Feign">
+                    <tag name="Execute"/>
+                    <tag name="Processing"/>
                     <tag name="Embedded"/>
```

Comment:
Here, the missing `</technology-identified>` tag is causing the build failure.
Even if, other than this, it looks like the `Feign` rule is replacing `Redis` one which I don't think it's the desired outcome but rather adding the `Feign` rule after the `Redis` one.